printer-linter/src/printerlinter/linters/formulas.py

```diff
@@ -0,0 +1,195 @@
+import difflib
+import json
+import os
+import re
+from configparser import ConfigParser
+from pathlib import Path
+from typing import Iterator
+
+from ..diagnostic import Diagnostic
+from ..replacement import Replacement
+from .linter import Linter
+
+FORMULA_NAMES = [
+    "extruderValue",
+    "extruderValues",
+    "anyExtruderWithMaterial",
+    "anyExtruderNrWithOrDefault",
+    "resolveOrValue",
+    "defaultExtruderPosition",
+    "valueFromContainer",
+    "extruderValueFromContainer",
+    "math",
+    "round",
+    "max",
+    "ceil",
+    "min",
+    "sqrt",
+    "log",
+    "tan",
+    "cos",
+    "sin",
+    "atan",
+    "acos",
+    "asin",
+    "pi",
```

Comment:
This is a constant so not callable as a function

Reply:
Removed all the non necessary constants, module or functions not used in printer definition here: https://github.com/Ultimaker/Cura/pull/18827/commits/72bf639beac129f6247a026d4c20cc56f2e2f372